DXMainClient/DXGUI/Generic/UpdateWindow.cs

```diff
@@ -289,8 +287,7 @@ private void CloseWindow()
             isStartingForceUpdate = false;
 
 #if WINFORMS
-            if (IsTaskbarSupported())
-                tbp.SetState(WindowManager.GetWindowHandle(), TaskbarProgress.TaskbarStates.NoProgress);
+            tbp.SetState(WindowManager.GetWindowHandle(), TaskbarProgress.TaskbarStates.NoProgress);
 
```

Comment:
While at it, could we get rid of these empty lines above the `#endif` directive? Could just be

```cs
#if WINFORMS
            tbp.SetState(WindowManager.GetWindowHandle(), TaskbarProgress.TaskbarStates.NoProgress);
#endif
```

Same for the other `#if WINFORMS` directives.